rpe/resources/gcp.py

```diff
@@ -81,7 +81,8 @@ def factory(resource_data, **kargs):
             'pubsub.projects.topics.iam': GcpPubsubTopicIam,
             'sqladmin.instances': GcpSqlInstance,
             'storage.buckets': GcpStorageBucket,
-            'storage.buckets.iam': GcpStorageBucketIamPolicy
+            'storage.buckets.iam': GcpStorageBucketIamPolicy,
+            'serviceusage.services': GcpProjectService
```

Comment:
Nitpicking, but can you throw this above the sqladmin.instances to keep them alphabetical by type?

Reply:
This has been moved.